allauth/socialaccount/providers/openid/views.py

```diff
@@ -34,7 +34,11 @@ def login(request):
             dict(list(request.GET.items()) + list(request.POST.items()))
         )
         if form.is_valid():
-            client = _openid_consumer(request)
+            provider = OpenIDProvider(request)
+            server_settings = \
+                provider.get_server_settings(form.cleaned_data['openid'])
+            stateless = server_settings.get('stateless', False)
```

Comment:
We're adding a new feature & setting here. Can you add this to the OpenID documentation over at providers.rst ?

Reply:
I've added docs now. Fairly basic but hopefully sufficient.